pkgs/applications/misc/bellepoule/default.nix

```diff
@@ -0,0 +1,91 @@
+{ lib
+, stdenv
+, fetchgit
+, goocanvas
+, pkg-config
+, gtk2
+, libxml2
+, curl
+, libmicrohttpd
+, libzip
+, libusb1
+, qrencode
+, json-glib
+, php82
+, libwebsockets
+, openssl
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Please use `finalAttrs` idiom instead of `rec`
https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes